README.md

```diff
@@ -21,3 +21,12 @@ Run models with `python3 evaluate_models.py [<number of cores to use>]`
 * Predictions saved to `results[_small]/predictions_<model_name>.csv`
 
 Score predictions with `python3 score_results.py results[_small]/<filename>.csv`
+
+## Creating a `conda` environment
+To create a `conda` virtual environment for running `example.py`, run the following commands:
+```
+conda create --name example cudatoolkit=9.0 tensorflow tensorflow-gpu cycler joblib kiwisolver matplotlib numpy pandas pyparsing python-dateutil pytz scikit-learn scipy six
+
+conda activate example
+pip install imbalanced-learn==0.6.2 sent2vec-prebuilt==0.0.3
```

Comment:
Can conda use the dependencies listed in `requirements.txt`?

Reply:
I ran into some errors because there are dependencies in our `requirements.txt` that have a version of `0.0.0` for some reason. `pip` handles this as "oh, that dependency is met" while `conda` says "oh no, can't find that version"
https://github.com/byu-dml/d3m-profiler/blob/9a3bc45061267091b0109f2159648785e370a18b/requirements.txt#L8
https://github.com/byu-dml/d3m-profiler/blob/9a3bc45061267091b0109f2159648785e370a18b/requirements.txt#L16

Reply:
Ah yes, I have seen this issue before. Both of those dependencies should be removed from `requirements.txt`. Can you remove them and then update the readme to use the requirements.txt?